tests/net/deckard/server.rs

```diff
@@ -86,34 +93,37 @@ fn do_adjust<Octs: Octets>(
         msg.header_mut().set_aa(true);
     }
     if reply.ad {
-        todo!()
+        msg.header_mut().set_ad(true);
```

Comment:
Same suggestion as above, call `header_mut()` once and then skip the if checks and just pass the `reply.bool` value to the appropriate setter.